pkg/network/cluster_config.go

```diff
@@ -116,9 +116,6 @@ func ValidateClusterConfig(clusterConfig *configv1.Network, client cnoclient.Cli
 		if infraRes.HostedControlPlane != nil {
 			return errors.Errorf("network type live migration is not supported on HyperShift clusters")
 		}
-		if !infraRes.StandaloneManagedCluster {
```

Comment:
I think there is no reason to keep the `StandaloneManagedCluster` detection if we want to remove the check.

Reply:
+1

Reply:
I'm just wondering if this information can be useful for other use-case in the future.

Reply:
Done